scripts/detect_missing_i18n.py

```diff
@@ -166,7 +150,7 @@ class Errtype(str, Enum):
 )
 
 i18n_element_missing_regex = opening_tag_syntax + ignore_after_opening_tag
-i18n_element_warn_regex = opening_tag_syntax + r"\$\((?!_\()"
+i18n_element_warn_regex = opening_tag_syntax + r"\$\(['\"]"
```

Comment:
Can you expand on the change here?

Reply:
Indeed! The fix here was to make the exclusion syntax specifically `$('` or `$("` rather than just `$(` because that was erroneously flagging a lot of python insertions into the HTML, i.e. `<div>$(page.title if page.type = "author" else "")</div>`, and because I think that was your original intended escape syntax 🙂